ormolu.cabal

```diff
@@ -98,10 +103,17 @@ library
         filepath >=1.2 && <1.5,
         directory ^>=1.3,
         Cabal >=3.6 && <3.7,
-        unordered-containers >=0.2 && <0.3,
-        hashable >=1.0 && <2.0,
         aeson >=1.0 && <3.0,
-        file-embed >=0.0.15 && <0.0.16
+        template-haskell,
+        th-lift-instances >=0.1 && <0.2
+
+    mixins:           ghc-lib-parser hiding (Language.Haskell.TH.Syntax)
```

Comment:
Could we do an import with explicit package name instead?

Reply:
Sure; I would have a very slight preference for using mixins as the disambiguation only happens in one place, this is how it would look like:

```diff
diff --git a/ormolu.cabal b/ormolu.cabal
index fec2f3f..bf21686 100644
--- a/ormolu.cabal
+++ b/ormolu.cabal
@@ -107,8 +107,6 @@ library
         template-haskell,
         th-lift-instances >=0.1 && <0.2
 
-    mixins:           ghc-lib-parser hiding (Language.Haskell.TH.Syntax)
-
     if flag(fixity-th)
         cpp-options: -DFIXITY_TH
 
diff --git a/src/Ormolu/Fixity.hs b/src/Ormolu/Fixity.hs
index 948583e..9636b34 100644
--- a/src/Ormolu/Fixity.hs
+++ b/src/Ormolu/Fixity.hs
@@ -1,6 +1,7 @@
 {-# LANGUAGE CPP #-}
 {-# LANGUAGE LambdaCase #-}
 {-# LANGUAGE MultiWayIf #-}
+{-# LANGUAGE PackageImports #-}
 {-# LANGUAGE TemplateHaskell #-}
 
 -- | Definitons for fixity analysis.
@@ -32,7 +33,7 @@ import qualified Data.Set as Set
 import Ormolu.Fixity.Types
 #if FIXITY_TH
 import Control.Monad.IO.Class (MonadIO (liftIO))
-import qualified Language.Haskell.TH.Syntax as TH
+import qualified "template-haskell" Language.Haskell.TH.Syntax as TH
 #else
 import Data.FileEmbed (embedFile)
 import Data.Maybe (fromJust)
diff --git a/src/Ormolu/Fixity/Types.hs b/src/Ormolu/Fixity/Types.hs
index a8e5bfb..16ba712 100644
--- a/src/Ormolu/Fixity/Types.hs
+++ b/src/Ormolu/Fixity/Types.hs
@@ -1,6 +1,7 @@
 {-# LANGUAGE DeriveLift #-}
 {-# LANGUAGE LambdaCase #-}
 {-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE PackageImports #-}
 {-# LANGUAGE RecordWildCards #-}
 {-# LANGUAGE StandaloneDeriving #-}
 
@@ -20,7 +21,7 @@ import Data.Map.Strict (Map)
 import Data.Text (Text)
 import qualified Data.Text as T
 import Instances.TH.Lift ()
-import qualified Language.Haskell.TH.Syntax as TH
+import qualified "template-haskell" Language.Haskell.TH.Syntax as TH
 
 -- | Fixity direction.
 data FixityDirection
```

Reply:
Okay, let's go with mixins :+1: